### ISSUE TITLE
Livereload not working on macOS with lima

### ISSUE THREAD

**Issue by Kidswiss** [vshn/antora-preview#8]
### Summary

Using docker via lima on macOS breaks the livereload.

### Context

Lima uses sshfs to map local folders to the docker VM and inotify doesn't seem to be working properly in that case.

### To Reproduce

* Start lima VM
* Configure Docker to use lima
* Start the antora-preview container
* Edit .adoc files
* The docker container doesn't register the changes and doesn't trigger a rebuild

### Workaround

Using guard locally with this configuraton:
```
group :documentation do
    # Rebuild documentation when modifying files
    guard :shell do
        watch(/(.*).adoc/) do
            `docker exec -it antora antora --cache-dir=/preview/public/.cache/antora /preview/playbook.yml`
        end
    end

    # Refresh browser when folder with HTML files changes
    guard :livereload do
        watch(/(.*).adoc/)
    end
end
```

Then run it:
```
cd project
guard &
docker run --name antora --rm --publish 35729:35729 --publish 2020:2020 --volume "${pwd}":/preview/antora -d vshn/antora-preview:2.3.12 --antora=docs --style=vshn
```

Not really sure how this could be solved though. Maybe just document this workaround somewhere? Also most people probably use Docker Desktop on Macs.

**Comment by akosma** [comment]
Not having a Mac to test this, so if there's a workaround for this issue, please add a paragraph in the README with exactly those steps, thanks!

**Comment by Kidswiss** [comment]
There's currently an open PR on the Lima repo that will change how host volumes are mapped: https://github.com/lima-vm/lima/pull/726

I'll test it once it's been merged, maybe inotify will work with this new method.

**Comment by tobru** [comment]
If inotify is the main issue, you could try `guard -p`.

```
Force usage of the Listen polling listener.
```

**Comment by LionsAd** [comment]
You need one of the various inotify forwarders to make that work even with 9pfs.

**Comment by khalo-sa** [comment]
Not 100% sure if you're dealing with the same issue as I did some weeks ago, but just in case. I'm using [Colima](https://github.com/abiosoft/colima/) (which is built on top of Lima) as an alternative to Docker Desktop, and for me Hot Reloading became really slow at some point. It took about 10-15 seconds for file changes on the host to be propagated into the container which was unbearable.
After a lot of research I found that SSHFS caching was introduced as a default configuration in Lima at some point, and I was able to get rid of that problem by creating the following override-file: 
```
# ~/.lima/_config/override.yaml
mounts:
  - location: "~"
    writable: true
    sshfs:
      cache: false
      followSymlinks: false
  - location: /tmp/colima
    writable: true
    sshfs:
      cache: false
      followSymlinks: false
```

Guess you have to recreate the Lima instance for changes to take effect. Hope it helps!

**Comment by akosma** [comment]
Solved with #10, thanks to @tobru for the tip!
